w3d_translator/w3d_translator.py

```diff
@@ -46,59 +39,42 @@ def farewell():
     exit(0)
 
 
-# Farewell message on error
-def farewell_error():
+# Run doctests
+def run_tests():
+    typer.echo("Running Tests")
```

Comment:
OOh a fun thing to do with tests is to also time them. If you're interested in optimizing and making things faster. No worries if not. I figured I'd toss it out there.

https://docs.python.org/3/library/timeit.html